src/pages/status/migration/index.jsx

```diff
@@ -176,13 +175,16 @@ function Filters({ counts, filters, onFilter }) {
           onClick={() => onFilter(key)}>
           {filters[key] ?
             <span className={[
-              styles[`${base}_on`],
-              styles.migration_details_filter_dot_on
+              styles[`${base}_hidden`],
+              styles.migration_details_filter_dot,
+              styles[`${base}_dot`],
             ].join(" ")}></span>
             :
             <span className={[
               styles[base],
-              styles.migration_details_filter_dot].join(" ")}>
+              styles.migration_details_filter_dot,
+              styles[`${base}_dot`],
```

Comment:
@jaimergp I may be wrong but it seems that line 185 and 186 are refering to the same class.